pages/collection.html

```diff
@@ -189,19 +189,36 @@ <h2 id="collected-cards-number">Collected cards:</h2>
       </section>
 
       <footer>
-        <a href="https://github.com/elysiumdelivery/altare-tcg"
-          ><svg
-            role="img"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
+        <div id="footer-nav">
+          <a id="footer-download-pdf" class="download-button" href="#">
```

Comment:
I assume the hrefs are blank intentionally since those aren't ready yet. I've made a separate task in Notion to add them when they're ready

Reply:
Yep, that's correct.